ghufran.js

```diff
@@ -0,0 +1,240 @@
+//1. Write a console.log statement saying "Hello World!"
+let greeting = "Hello, world!";     // English
+console.log(greeting);
+
+greeting = "Hallo, wereld!";     // Dutch
+console.log(greeting);
+
+greeting = "Marhaba, Aalam!";     // Arabic
+console.log(greeting);
+
+greeting = "Halo, dunia!";      // Indonesian
+console.log(greeting);
+
+greeting = "Ciao, mondo!";      // Italian
+console.log(greeting);
+
+greeting = "Hola, mundo!";       // Spanish
+console.log(greeting);
+
+console.log("---------------");
+
+//2. Consider the following code:
+console.log('I\'m awesome');
+
+console.log("---------------");
+
+//3. Declare a variable x and initialize it with an integer
+let x;    
+console.log("the value of my variable x will be: undifined");
+console.log(x);
+
+x = 21;     //const x = 21;     //The output is error, I cann't use initialization, I have to use difination.
+console.log("the value of my variable x will be: 21");
+console.log(x);
+
+console.log("---------------");
+
+//4. Declare a variable y and assign a string to it.
+let y = "I'm trying to improve my typing skills";   
+console.log("The value of my variable y will be: I'm trying to improve my typing skills");
+console.log(y);
+
+y = "It is more difficult than it seems to be";
+console.log("The value of my variable y will be: It is more difficult than it seems to be");
+console.log(y);
+console.log("---------------");
+
+//5. How do you round the number 7.25, to the nearest integer 
+const z = 7.25;
+console.log(z);
+
+const a = 7;
+console.log(a);
+
+const highestValue = Math.max(7, 7.25);
+
+console.log(highestValue);
+
+console.log("---------------");
+
+//6. Arrays
+const myEmptyValues = [];
+console.log("The value of my array will be: []");   //because it has no given value, it will be empty.... not undefined because it is not a value 
+console.log(myEmptyValues);
+
+const myFavouriteAnimals = ["Cats", "Birds", "Rabbits"];
+console.log(myFavouriteAnimals);
+
+myFavouriteAnimals.push("baby pig");
+console.log(myFavouriteAnimals);
+
+console.log("---------------");
+
+//7. More strings
+let myString = "this is a test";
+console.log(myString);
+
+myStringLength = myString.length;
+console.log(myStringLength);
```

Comment:
Ah. This is a bug here. You haven't declared `myStringLength` here using the `const` or `let` or `var` keyword and are trying to assign it a value straightaway. We would be covering this in our class tomorrow. But for now, please remember to declare a variable using one of the three keywords before trying to use it for reading/writing.